src/base.jl

```diff
@@ -2341,7 +2341,11 @@ function get_bus_numbers(sys::System)
     return sort(collect(sys.bus_numbers))
 end
 
+_fetch_match_fn(match_fn::Function) = match_fn
```

Comment:
Can't you call the IS method instead of re define?

Reply:
Almost did, but I was thinking it didn't make sense to add `_fetch_match_fn` to IS's public interface and didn't want to rely on private stuff

Reply:
Not a fan of cross repo duplication but it's a minimal thing